packages/common/src/api/GqlContext.tsx

```diff
@@ -123,7 +119,10 @@ interface ApiProviderProps {
   url: string;
 }
 
-export const GqlProvider: FC<ApiProviderProps> = ({ url, children }) => {
+export const GqlProvider: FC<PropsWithChildren<ApiProviderProps>> = ({
```

Comment:
and a _lot_ of these. React now requires explicit declaration of support for `children` ( previously using `React.FC` included a children prop )